client/components/table/style.scss

```diff
@@ -34,6 +34,7 @@
 
 .woocommerce-table__header,
 .woocommerce-table__item {
+	font-size: 0.8125rem;
```

Comment:
could we use the font-size mixin here? https://github.com/woocommerce/wc-admin/blob/7555ce5bb6e4f135bd6a5766915bcc1591434fe8/client/stylesheets/abstracts/_mixins.scss#L4

Reply:
Absolutely!